prelude/__internal__/List.mad

```diff
@@ -154,7 +156,7 @@ export append = (v, xs) => (#- [...xs, v] -#)
 
 
 /**
- * Returns a Maybe containing either a Just with the last element or a Nothing if the list is empty.
+ * Safely access the last value in a List (wrapped in Just), or Nothing.
```

Comment:
I liked in the previous version the precision of when a Nothing can happen.

Reply:
How do you feel about: "Safely access the last value in a List (wrapped in Just), or Nothing if the list is empty."

Reply:
Amended to the above.